ui/src/LandingPage.js

```diff
@@ -423,6 +452,15 @@ class LandingPage extends Component {
     });
   };
 
+  onNewTemplateMetadataChange = (event) => {
+    const resetValidation = !event.target.value.trim();
+
+    this.setState({
+      newTemplateMetadata: event.target.value,
+      newTemplateMetadataValidationError: resetValidation,
```

Comment:
This highlights empty metadata as an error, I think it should just set `newTemplateMetadataValidationError: false` since it is allowed to be empty?